docs/terms.html

```diff
@@ -167,16 +173,25 @@ <h3 class="govuk-heading-s">Version effective from 23rd March 2020.</h2>
         <u>Linking from Street Manager via APIs</u>
       </p>
       <p class="govuk-body">
-        Street manager can be linked to other websites and services via an API.  The DfT does not have any control over the content on these websites or services.  It is therefore not responsible for the security, performance or reliability of these other websites or services.
+        Street manager can be linked to other websites and services via an API. The DfT does not have any control over the content on these websites or services. It is therefore not responsible for the security, performance or reliability of these other websites or services.
       </p>
       <p class="govuk-body">
         Users should read all terms and conditions, privacy policies and end user licences that relate to these websites or services before you use them.
       </p>
+      <p class="govuk-body">
+        The primary contact and/or administration users should ensure that any integrators they allow to use Street Manager on their behalf are aware of these terms and conditions.
+      </p>
+      <p class="govuk-body">
+        The DfT reserves the right to impose limits on the Street Manager API service based upon the monitored use and performance metrics of the system. We may contact administration users and representatives from the integrators if our automated tooling determines issues with an integration.
```

Comment:
would monitoring tools be a better that tooling
```suggestion
        The DfT reserves the right to impose limits on the Street Manager API service based upon the monitored use and performance metrics of the system. We may contact administration users and representatives from the integrators if our automated monitoring tools determine issues with an integration.
```